File: problemsKylePade/problemsKylePade_283/solution.go
package problem283

import (
	"encoding/json"
	"log"
	"strings"
)

func moveZeroes(nums []int)  {
    
}

func Solve(inputJsonValues string) interface{} {
	inputValues := strings.Split(inputJsonValues, "\n")
	var nums []int

	if err := json.Unmarshal([]byte(inputValues[0]), &nums); err != nil {
		log.Fatal(err)
	}

	moveZeroes(nums)
	return nums
}

File: problemsKylePade/problemsKylePade_283/Solution.java
package problemsKylePade.problemsKylePade_283;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public void moveZeroes(int[] nums) {
        
    }

    @Override
    public Object solve(String[] inputJsonValues) {
        int[] nums = jsonArrayToIntArray(inputJsonValues[0]);
		moveZeroes(nums);
        return JSON.toJSON(nums);
    }
}
